src/api/app/views/shared/_requests.html.erb

```diff
@@ -3,7 +3,7 @@
   request_table_id ||= 'request_table'
   hide_state ||= false
   no_target ||= false
-  paginate = serverProcessing ? true : requests.count > 25
+  paginate = serverProcessing || (!requests.nil? && requests.count > 25)
```

Comment:
Ah, ok. But there isn't any has many relation between users and requests... It's BsRequestCollection used here. And sometimes become nil for some users.